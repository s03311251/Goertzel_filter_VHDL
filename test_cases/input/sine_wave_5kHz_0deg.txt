2000
2101
2202
2302
2402
2501
25FE
26FA
27F5
28ED
29E3
2AD6
2BC7
2CB5
2D9F
2E86
2F6A
3049
3125
31FC
32CE
339C
3465
3529
35E7
36A0
3753
3800
38A7
3948
39E3
3A77
3B04
3B8A
3C0A
3C82
3CF3
3D5D
3DC0
3E1B
3E6E
3EBA
3EFE
3F3A
3F6E
3F9A
3FBE
3FDB
3FEF
3FFB
3FFF
3FFB
3FEF
3FDB
3FBE
3F9A
3F6E
3F3A
3EFE
3EBA
3E6E
3E1B
3DC0
3D5D
3CF3
3C82
3C0A
3B8A
3B04
3A77
39E3
3948
38A7
3800
3753
36A0
35E7
3529
3465
339C
32CE
31FC
3125
3049
2F6A
2E86
2D9F
2CB5
2BC7
2AD6
29E3
28ED
27F5
26FA
25FE
2501
2402
2302
2202
2101
