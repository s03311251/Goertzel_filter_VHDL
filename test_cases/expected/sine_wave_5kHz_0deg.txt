FFFFFFE0
00001460
